spec/index.html

```diff
@@ -873,6 +859,7 @@ <h4>The <code>&lt;<dfn id="the-layer-element">layer</dfn>&gt;</code> element</h4
           if the layer is not not <a href="#attr-layer-hidden"><code>hidden</code></a>. The map author's <a href="#the-title-element"><code>title</code></a> element takes precedence over <code>label</code>, if available.</dd>
```

Comment:
the label is not linked to the hidden attribute anymore

Reply:
Yes, there's the "not not" which is a typo, but the concept of layers being hidden due to not having a label is a confusion with the old implementation of `map-extent`.  Layers were never hidden for that reason.